src/main/resources/org/jenkinsci/plugins/envinject/EnvInjectBuildWrapper/help-propertiesContent.html

```diff
@@ -0,0 +1,6 @@
+<div>
```

Comment:
I just mean it makes sense to put the extended help message to resources and reference it explicitly